app/controllers/katello/api/v2/systems_controller.rb

```diff
@@ -388,8 +388,13 @@ def setup_content_view(cv_id)
     end
 
     def systems_by_errata(errata_uuids, installable, non_installable)
-      installable = installable.nil? ? false : installable.to_bool
-      non_installable = non_installable.nil? ? false : non_installable.to_bool
+      if !!installable != installable
+        installable = installable.nil? ? false : installable.to_bool
+      end
+
+      if !!non_installable != non_installable
+        non_installable = non_installable.nil? ? false : non_installable.to_bool
+      end
```

Comment:
The 'conversion' is needed because the use can pass a list of errata_ids and get three different responses based on other parameters:

* With no other parameters, should return list of systems with those errata applicable
* passing erratum_restrict_installable set to true would only return systems with those errata installable
* passing erratum_restrict_non_installable returns the list of systems where the provided errata are NOT installable.